release/models/interfaces/openconfig-if-ip.yang

```diff
@@ -605,6 +620,16 @@ revision "2023-06-30" {
         "RFC 4862: IPv6 Stateless Address Autoconfiguration";
     }
 
+    leaf unsolicited-na-accepted {
+      type boolean;
+      default true;
```

Comment:
since this leaf is supposed to enable 9131 behavior only, please remove the default

Reply:
Removed the default value